eternalcore-core/src/main/java/com/eternalcode/core/EternalCore.java

```diff
@@ -300,11 +293,19 @@ public EternalCore(Plugin plugin) {
         LanguageInventory languageInventory = new LanguageInventory(languageConfiguration, this.noticeService, this.userManager, this.miniMessage);
         WarpInventory warpInventory = new WarpInventory(this.teleportTaskService, this.translationManager, this.warpManager, this.miniMessage);
 
+        this.bridgeManager = new BridgeManager(this.placeholderRegistry, server, plugin.getLogger());
+        this.bridgeManager.init();
+
         /* Frameworks & Libraries */
         this.skullAPI = LiteSkullFactory.builder()
             .bukkitScheduler(plugin)
             .build();
 
+        this.placeholderRegistry.registerPlaceholder(PlaceholderReplacer.of("online", player -> String.valueOf(server.getOnlinePlayers().size())));
+        // Tutaj jest problem i nie wiem jak to naprawić
```

Comment:
Jaki problem

Reply:
Wywala błąd `Option has not been completed`

Reply:
rel, proszę to naprawić

Reply:
> > Wywala błąd `Option has not been completed`
> 
> no bo z bomby dajesz Completable#get

no a jak to naprawić bo nie mam pomysłu

Reply:
Upewniaj się że Completable zostało wykonane i dopiero wtedy przekazuj wartość

Reply:
At the moment I don't see a way to solve this problem, although you can use then/thenApply, but then it is not possible to transfer the player, at the moment I am of the opinion that the PlaceholderAPI system has been completed, and adding the rest of the placeholders should be transferred to other **PRs**, Two simple placeholders have been added, so this is enough for a start. Another solution is possible, but I think that it would be a bit more complicated, I move this to another PR.